entrypoint.sh

```diff
@@ -1,8 +1,9 @@
 #!/bin/bash -ue
 
-repo=$1
-file=$2
-output=keybase-secret-${file/\//-}
+repo="${1}"
+file="${2}"
+prefix="${3:-keybase-secret-}"
```

Comment:
to allow setting prefix to an empty string, we must use

```
prefix="${3:keybase-secret-}"
```
instead of
```
prefix="${3:-keybase-secret-}"
```